### ISSUE TITLE
Invalid dataloader, please check if the input dataloder is valid.

### ISSUE THREAD

**Issue by kamathis4** [TorchEnsemble-Community/Ensemble-Pytorch#90]
Tried to use multi-label classifier with Gradient boosting. It works with normal pretrained models well enough. I don't know what's the problem here

**Comment by xuyxu** [comment]
Hi @kamathis4, currently multi-label classification is not supported ;-(
